styles.css

```diff
@@ -181,4 +704,170 @@ ul.navbar-items {
     background-size: 250px;
     background-repeat: no-repeat;
   }
+
+  /* works-section responsive style  */
+
+  #works-section {
+    margin-bottom: 100px;
+  }
+
+  .box-container {
+    flex-direction: column;
+    margin: 5px 10px;
+  }
+
+  .left-section {
+    margin-bottom: 20px;
+  }
+
+  .left-section img {
+    width: 100%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 10px;
+    margin: 0;
+    padding-left: 4%;
+    padding-right: 4%;
+  }
+
+  .first-desc {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .first-desc h3 {
+    font-size: 16px;
+  }
+
+  .first-desc ul {
+    padding-left: 0;
+  }
+
+  .first-desc p {
+    font-size: 14px;
+    text-align: left;
+  }
+
+  .first-desc li {
+    display: inline-block;
+    margin: 10px 0;
+    padding-left: 10px;
+  }
+
+  .hr {
+    margin: 20px;
+  }
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 use the correct font family and styles to be similar to the Figma template 👇🏽

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Xghayor/Portfolio/assets/58642949/d81ba763-883d-4a5a-9088-6576945a4416) | ![image](https://github.com/Xghayor/Portfolio/assets/58642949/5109aa6d-ac00-4fea-9de9-5ed74e3945a5) |